### PR TITLE
Moving example files to the right subfolder, fixing CMakeLists

### PR DESCRIPTION
**Pull request by pclausen**
Moving example files to the right subfolder, fixing CMakeLists, config.json and ci accordingly

Now build works, ci may need some work. 

I would like to add windows but I need to build on windows using MingW toolchain. @ErikSchierboom do you know someone who may be able to help ?

### CODE REVIEW

**Comment by pclausen** [comment]
When this is merged I will fix PR #103 as well

File: config.json [hunk 1]
```diff
@@ -11,10 +11,10 @@
     "analyzer": false
   },
   "files": {
-    "solution": ["%{snake_slug}.f90"],
+    "solution": [".meta/example.f90"],
```
Comment (ErikSchierboom) [inline config.json:14]: I think you updated the wrong file here. You probably want to update the `example` key.

**Review by ErikSchierboom — COMMENTED** [review]
It looks like the actual run command is the same across platforms. You could consider using the `matrix` functionality that allows you to DRY this up a bit: https://github.com/exercism/configlet/blob/main/.github/workflows/tests.yml#L16

You could also do this later.

File: config.json (hunk 1 above)
Comment (pclausen) [reply config.json:14]: Hmm, OK got confused but I reread: [https://exercism.lol/docs/building/tracks/config-json](https://exercism.lol/docs/building/tracks/config-json). I will fix accordingly.

So "solution" is the file that the exercism-user will use as template for creating their solution. I think work "template"  would be better in my mind, but OK. 
"example" is then the solution or the actual implementation of the exercise, which I thought was required but seems to be optional in doc. What is "examplar" and "editor"?

**Comment by pclausen** [comment]
@ErikSchierboom Ha, actually got mac up and running after too much googling. I would like to get windows running as well but I need to install mingw32-gfortran and I do not know how to do that. Also, as you suggested I should clean up using matrix functionality, but I have not yet read up on how to do that.

**Comment by pclausen** [comment]
Adding windows was much easier than I thought. 

> You could consider using the matrix functionality

I actually do not understand what it does or why I should use it. For now I will leave this as it is.

**Comment by ErikSchierboom** [comment]
> I actually do not understand what it does or why I should use it. For now I will leave this as it is.

That's perfectly fine. What it does is allow you to define the various test hosts (Windows/macOS/Unix) in once place, and have them all execute the same path (with host-specific options). But it's fine to have it duplicated.

**Review by ErikSchierboom — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -7,12 +7,31 @@ on:
...
+
+  linux-latest:
+    name: Linux Latest
+    needs: [linux-min]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
       - name: Install project dependencies
         run: |
           sudo apt-get update -q
```
[the remark this answers is not in the document]
Comment (pclausen) [reply .github/workflows/ci.yml]: agreed, removed

**Pull request merged**